extra-dev/packages/coq-equations/coq-equations.dev/opam

```diff
@@ -32,6 +32,9 @@ run-test: [
 depends: [
   "coq" {= "dev"}
 ]
+depopts: [
+  "coq-hott" {= "dev"}
```

Comment:
I do not think `depopts` means what you think it means. By writing `{= "dev"}`, you are telling Opam that Hott is an optional dependency of Equations only if it is at the `dev` version. So, if the user has another version, then Hott will not be marked as a dependency, yet `--enable-hott` will still be passed to `configure.sh`.

If you really need to depend on the `dev` version of Hott (rather than any version of Hott compatible with Coq `dev`), you need to write something like
```
depopts: [ "coq-hott" ]
conflicts: [ "coq-hott" {<> "dev"} ]
```

Reply:
Right, thanks